docs/examples/DataSet/Measuring X as a function of time.ipynb

```diff
@@ -0,0 +1,179 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Measuring X as a function of time\n",
+    "\n",
+    "Sometimes we'd like to measure something as a function of elapsed wall clock time. This notebook provides a simple example of doing that using the `ElapsedTimeParameter`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "import numpy as np\n",
+    "\n",
+    "from qcodes.utils.time import ElapsedTimeParameter\n",
+    "from qcodes.instrument.parameter import Parameter\n",
+    "from qcodes.dataset.measurements import Measurement\n",
+    "from qcodes.dataset.sqlite.database import initialise_or_create_database_at\n",
```

Comment:
i'm not yet comfortable with exposing where this function comes from. i prefer that for now it comes just from `qcodes.`, otherwise people will start looking into what else they can use from `.sqlite.*` while they are not allowed because it's all our internal stuff that can change, etc.

Reply:
I have moved one import to resolve this. We should make a separate PR that attacks the mess of the qcodes namespace.